use_cases/get_all_bookings_by_pro_user.py

```diff
@@ -0,0 +1,14 @@
+from typing import List
+
+from domain.booking_recap.booking_recap import BookingRecap
+from domain.users import check_user_is_not_admin
+from repository import user_queries, booking_queries
+
+
+def get_all_bookings_by_pro_user(user_id: int) -> List[BookingRecap]:
```

Comment:
```suggestion
def get_all_bookings_by_pro_user(user_id: int, find_user_by_id: Callable) -> List[BookingRecap]:
```
Voici un exemple de ce que l'on a fait : https://github.com/betagouv/pass-culture-api/pull/1196/files#diff-8290d842f68106a3e0c01bab361f8f33R7